### PR TITLE
[PANW] add audit log parsing

### PR DESCRIPTION
**Pull request by gogochan**
## Proposed commit message

Explain here the changes you made on the PR.

Please explain:
This PR adds Audit log, described in https://docs.paloaltonetworks.com/pan-os/11-0/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/audit-log-fields, processing for PaloAlto's NG firewall.

The audit log has different log pattern than the existing ones. So I added a new grok pattern to match the log provided by the customer. I've asked customer to provide additional sample log if possible. The PR may be updated based on additional information I get from the customer.

The issue was reported by https://github.com/elastic/sdh-beats/issues/4641

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

-

### CODE REVIEW

File: packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/audit.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      ignore_failure: true
+
+on_failure:
+  - set:
+      field: event.kind
+      value: pipeline_error
+  - append:
+      field: error.message
+      value: >-
+        error in IP Tag pipeline:
```
Comment (taylor-swanson) [inline packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/audit.yml:50]:
```suggestion
        error in Audit pipeline:
```

**Review by taylor-swanson — COMMENTED** [review]
LGTM aside from one issue

File: packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/audit.yml (hunk 1 above)
Comment (gogochan) [reply packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/audit.yml:50]: Thanks! 🙏

**Review by taylor-swanson — APPROVED** [review]

**Pull request merged**